src/yunohost/service.py

```diff
@@ -639,12 +639,17 @@ def _get_pending_conf(services=[]):
 
 def _get_conf_hashes(service):
     """Get the registered conf hashes for a service"""
-    try:
-        return _get_services()[service]['conffiles']
-    except:
-        logger.debug("unable to retrieve conf hashes for %s",
-                     service, exc_info=1)
+
+    d = _get_services()
```

Comment:
Maybe I'm old-fashion, but I would prefer an explicit variable name, like `services`, because `d` does not mean anything to me ;)